docs/installation.rst

```diff
@@ -34,57 +34,79 @@ A possible (default) output can be ``~/.local/bin/esm_tools``.
 Installing using pip
 --------------------
 
-tbd
+There is a version of `ESM-Tools` available that can be installed using ``pip``:
+    
+- Update ``pip`` and ``wheel`` python packages::
+    
+    $ pip install -U pip wheel
+    
+- Install `ESM-Tools`::
+    
+    $ pip install esm-tools 
+
+.. note::
+   The version that is available via ``pip`` is not the most recent version. We strongly recommend to install the most recent version of `ESM-Tools` using the install script described above (see also :ref:`get_esm-tools:Get ESM-Tools` and :ref:`installation:Installing in local environment`).
```

Comment:
I think this version doesn't work, or if it works, one cannot easily find the configuration files. Can you please check that that's the case? If yaml files are difficult to find, maybe we should consider removing this.